lib/asana/http_client/error_handling.rb

```diff
@@ -28,7 +30,7 @@ module ErrorHandling
       # Raises [Asana::Errors::APIError] when the API returns an unknown error.
       #
       # rubocop:disable all
-      def handle(&request)
+      def handle(num_timeouts=0, &request)
```

Comment:
Several other client libraries make automatic retries optional, and allow the user to set how many times they want to try. What would the estimated effort to have this function more like [this](https://github.com/Asana/python-asana/blob/master/asana/client.py#L105)?